README.md

```diff
@@ -292,19 +291,16 @@ spec:
         level: "default:warning" # override from info
 ```
 
-Since from 1.3 Helm charts are split up per component, values overrides should be specified under the appropriate component
+Values overrides can also be specified for a particular component
  ([samples/values-pilot.yaml](samples/values-pilot.yaml)):
 
 ```yaml
 apiVersion: install.istio.io/v1alpha2
 kind: IstioControlPlane
 spec:
-  trafficManagement:
-    components:
-      pilot:
-        common:
-          values:
-            traceSampling: 0.1 # override from 1.0
+  values:
+    pilot:
+      traceSampling: 0.1 # override from 1.0
```

Comment:
Not sure if this example is suitable here, because i have tried this case, looks like it doesn't work:
```
# out/bin/mesh manifest generate -f samples/pilot-k8s.yaml | grep -i sampling
        - name: PILOT_TRACE_SAMPLING
    traceSampling: 1
```

As you can see `traceSampling` isn't overrided, because we have following k8s env overlay in `default` profile:
```
          k8s:
            env:
              - name: POD_NAME
                valueFrom:
                  fieldRef:
                    apiVersion: v1
                    fieldPath: metadata.name
              - name: POD_NAMESPACE
                valueFrom:
                  fieldRef:
                    apiVersion: v1
                    fieldPath: metadata.namespace
              - name: GODEBUG
                value: gctrace=1
              - name: PILOT_TRACE_SAMPLING
                value: "1"
              - name: CONFIG_NAMESPACE
                value: istio-config
```

The current workaround is override `traceSampling` is update it in k8s overlay:
```
apiVersion: install.istio.io/v1alpha2
kind: IstioControlPlane
spec:
  trafficManagement:
    components:
      pilot:
        common:
          k8s:
            env:
              - name: POD_NAME
                valueFrom:
                  fieldRef:
                    apiVersion: v1
                    fieldPath: metadata.name
              - name: POD_NAMESPACE
                valueFrom:
                  fieldRef:
                    apiVersion: v1
                    fieldPath: metadata.namespace
              - name: GODEBUG
                value: gctrace=1
              - name: PILOT_TRACE_SAMPLING
                value: "0.1"
              - name: CONFIG_NAMESPACE
                value: istio-config
```

Reply:
Done.